llpc/context/llpcCompiler.cpp

```diff
@@ -2172,6 +2179,8 @@ void Compiler::LinkRelocatableShaderElf(
                 return;
             }
         }
+        WriteElfFile("vs.elf", pShaderElfs[ShaderStageVertex]);
+        WriteElfFile("fs.elf", pShaderElfs[ShaderStageFragment]);
```

Comment:
We need to remember to make this conditional and more robust.  That is, what about compute pipelines?  No need to fix that for now.